ansible_collections/arista/avd/docs/contribution/overview.md

```diff
@@ -65,6 +65,15 @@ Should this happen, and if you followed the project guidelines, have sample test
 
 ### Contributing to Arista Validated Design
 
+Contributing pull requests are gladly welcomed for this repository. If you are planning a big change, please start a discussion first to make sure we'll be able to merge it.
+
+#### Branching model
+
+- The **`devel`** branch corresponds to the release actively under development.
+- The **`releases/x.x.x`** branches correspond to stable releases.
```

Comment:
This is not true. We use tags and not branches for releases.

Reply:
Will update; we had just moved this content from the GitHub/Readme.md